core/src/main/java/org/mskcc/cbio/portal/dao/DaoGeneticEntity.java

```diff
@@ -0,0 +1,55 @@
+package org.mskcc.cbio.portal.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class DaoGeneticEntity {
+
+	/**
+     * Private Constructor to enforce Singleton Pattern.
+     */
+    private DaoGeneticEntity() {
+    }
+    
+    public static enum EntityTypes
```

Comment:
@ersinciftci said these embedded enum classes are still globally usable (constants not restricted in scope to the contained methods).
I do wonder whether the enum constant "GENE" is likely to clash with enum constants in other locations. This may lead to confusion. I would suggest constants such as GENETIC_ENTITY_TYPE_GENE, and GENETIC_ENTITY_TYPE_GENE_SET. This is my gut feeling .. perhaps it is not needed.